ui/Surface.js

```diff
@@ -770,6 +777,16 @@ class Surface extends Component {
     window.setTimeout(fn, BROWSER_DELAY)
   }
 
+  // prevent the native behavior of key combos
+  _muteNativeHandlers(event) {
+    const key = parseKeyEvent(event)
+    const keyboardManager = this.editorSession.keyboardManager
+
+    if(keyboardManager.keydownBindings[key]) {
```

Comment:
This is maybe a bit implicit. What about adding a switch-case block where we add the keys excplicitly, then everybody understands what we do there.

Reply:
And I think, we should only suppress default ContentEditable behavior here.
E.g. `CMD-F`  should be suppressed maybe somewhere else.